HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 ---
+## 1.5.0 / 2017-02-10
+- add `onDoubleClick` API
+- add `toggleSelect` option for changing how selection works
```

Comment:
I haven't looked at the code in a bit so I'm not 100% sure, but the toggleSelect will allow clicks to toggle the selected state (I believe that is the behaviour now), with it disabled clicks will always select the node and not toggle. I believe your suggestion will disable selection altogether?